regression-tests/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "@autorest/regression-test",
```

Comment:
Since this is technically a package, should it be under `packages` somewhere?  Maybe a `packages/testing` or `packages/tools`

Reply:
Hhm, well it isn't a package linked to rush and really isn't meant to be published or consumed